tests/target/issue-4546.rs

```diff
@@ -0,0 +1,165 @@
+// Basic tests (using vectors) - list multiline post-comment is indented on its own
+fn main() {
+    let v = [
+        "A",   /* item A comment */
+        "BBB", /* item B comment */
+        "CCCCCC", /* item C comment line 1
+                   * item C comment line 2 */
+        "D", /* item D comment line 1
+              * item D comment line 2 */
+        "EEEEE", /* item E comment */
+        "FFF",   /* item F comment */
+        "GG", /* item G comment line 1
+               * item G comment line 2 */
+    ];
+}
+
+fn main() {
+    let v = [
+        "GG", /* item G comment line 1
+               * item G comment line 2 */
+        "AAAAA", /* item A comment */
+        "BBB",   /* item B comment */
+        "CCCCCC", /* item C comment line 1
+                   * item C comment line 2 */
+        "D", /* item D comment line 1
+              * item D comment line 2 */
+        "E",   /* item E comment */
+        "FFF", /* item F comment */
+    ];
+}
+
+// Tests for Struct
+pub(crate) struct ListFormatting<'a> {
+    tactic: DefinitiveListTactic,
+    separator: &'a str,                  /* Comment */
+    trailing_separator: SeparatorTactic, /* Comment */
+    separator_place: SeparatorPlace,     // Comment 1
```

Comment:
should these comments be aligned?

Reply:
The reason for this formatting is backward compatibility - see [this comment](https://github.com/rust-lang/rustfmt/issues/4546#issuecomment-736712067).  As formatting of multi-line comments changes, is it better to give up the backward compatibility for one line comments alignment?

Reply:
Thanks for the link to the previous discussion!